src/main/java/de/hpi/bpt/argos/api/eventType/EventTypeEndpointImpl.java

```diff
@@ -1,93 +1,425 @@
+
+            for (EventQuery query : queries) {
+                JsonObject jsonTypeAttribute = new JsonObject();
+
+                jsonTypeAttribute.addProperty("Id", query.getId());
+                jsonTypeAttribute.addProperty("Description", query.getDescription());
+                jsonTypeAttribute.addProperty("Query", query.getQuery());
+
+                jsonTypeQueries.add(jsonTypeAttribute);
+            }
+        } catch (Exception e) {
+            logger.error("cannot parse request body to event queries '" + request.body() + "'");
+            logTrace(e);
+            halt(HttpStatusCodes.ERROR, e.getMessage());
+        }
+        return serializer.toJson(jsonTypeQueries);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getEventTypeEntityMappings(Request request, Response response) {
+        endpointUtil.logReceivedRequest(logger, request);
+        JsonArray jsonEntityMappings = new JsonArray();
+
+        try {
+            long eventTypeId = getEventTypeId(request);
+            List<EventEntityMapping> entityMappings = PersistenceAdapterImpl.getInstance().getEventEntityMappingsForEntityType(eventTypeId);
+
+            for (EventEntityMapping entityMapping : entityMappings) {
+                JsonObject jsonEntityMapping = new JsonObject();
+
+                jsonEntityMapping.addProperty("Id", entityMapping.getId());
+                jsonEntityMapping.addProperty("EventTypeId", entityMapping.getEventTypeId());
+                jsonEntityMapping.addProperty("EntityTypeId", entityMapping.getEntityTypeId());
+                jsonEntityMapping.addProperty("TargetStatus", entityMapping.getTargetStatus());
+
+                // add mapping conditions as array
+                JsonArray jsonMappingConditions = new JsonArray();
+                List<MappingCondition> mappingConditions = PersistenceAdapterImpl.getInstance()
+                        .getMappingConditionsForMapping(entityMapping.getId());
+                for (MappingCondition condition : mappingConditions) {
+                    JsonObject jsonCondition = new JsonObject();
+                    jsonCondition.addProperty("EventTypeAttributeId", condition.getEventTypeAttributeId());
+                    jsonCondition.addProperty("EntityTypeAttributeId", condition.getEntityTypeAttributeId());
+                    jsonMappingConditions.add(jsonCondition);
+                }
+                jsonEntityMapping.add("EventEntityMappingConditions", jsonMappingConditions);
+
+                jsonEntityMappings.add(jsonEntityMapping);
+            }
+        } catch (Exception e) {
+            logger.error("cannot parse request body to event entity mappings '" + request.body() + "'");
+            logTrace(e);
+            halt(HttpStatusCodes.ERROR, e.getMessage());
+        }
+        return serializer.toJson(jsonEntityMappings);    }

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)